src/data/showcases.js

```diff
@@ -268,11 +268,20 @@ export const Showcases = [
     tags: ["explorer"],
   },
   {
-    title: "Cardano Explorer",
+    title: "Cardano Foundation Explorer",
     description:
-      "The Cardano explorer built by IOHK, one of the founding entities of Cardano.",
-    preview: require("./showcase/cardanoexplorer.png"),
-    website: "https://explorer.cardano.org",
+      "A Cardano explorer focused on the needs of a non technical target audience like regulators and policy makers built by Cardano Foundation, currently under development",
+    preview: require("./showcase/cf-explorer.png"),
+    website: "https://beta.explorer.cardano.org",
```

Comment:
what are the plans/path to move from beta.explorer.cardano.org to explorer.cardano.org?

Reply:
We are figuring this out as we speak. What is clear already is that what is currently running on explorer.cardano.org will not make it to 9.1. 

Options on the table atm: 
- Replace it
- Landing page on explorer.cardano.org who lists other explorers. (That someone needs to maintain)
- Redirect to showcase with the explorer category. 

Any opinions welcome.

Reply:
Replace it =] 
A landing page for explorers could be with plural 😉

Reply:
You are not wrong sir.